src/plugins/formulas/__tests__/formulas.spec.js

```diff
@@ -1595,6 +1663,36 @@ describe('Formulas general', () => {
         ['y'], ['=A3'], ['=A6'], ['=A5'], ['=A8'], ['=A7'], ['=A10'], ['=A9'], ['=A12']
       ]);
     });
+
+    it('autofill suports validation', async() => { 
+      const hot = handsontable({ 
+        data: [ 
+          [1, 2, 3],
+          ['txt', 1, 2], 
+    ], 
+      formulas: { 
+        engine: HyperFormula
+       }, 
+      columns: { 
+        type: 'numeric' 
+      }, 
```

Comment:
What do you mean by that? There should be like that:
```suggestion
      columns: [ 
        { type: 'numeric' },
        {}
      ], 
```